src/utils/multivariate/multivariateStyle.ts

```
@@ -0,0 +1,127 @@
+    paint: {
+      'fill-color': setupColorClasses(
+        baseValueExpression,
+        annesValueExpression,
+        DEFAULT_MULTIBIVARIATE_STEPS,
+        DEFAULT_MULTIBIVARIATE_STEPS,
+        colorsMap(colors),
+      ),
+      'fill-opacity': 1,
+      'fill-antialias': false,
+    },
+    source: id + '_source', // this id is replaced inside the Renderer
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)